bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/values/BRegexpValue.java

```diff
@@ -28,6 +28,8 @@
  */
 public interface BRegexpValue extends BValue {
 
+    // TODO: remove this with https://github.com/ballerina-platform/ballerina-lang/issues/40175
+    @Deprecated(since = "2201.7.0", forRemoval = true)
     public RegExpDisjunction getRegExpDisjunction();
```

Comment:
```
    // TODO: remove this with https://github.com/ballerina-platform/ballerina-lang/issues/40175
    @Deprecated(since = "2201.7.0", forRemoval = true)
    public RegExpDisjunction getRegExpDisjunction();
```
`RegExpDisjunction` is `runtime.internal.values.RegExpDisjunction`. @pcnfernando is it okay to deprecate and remove this later? Or do we need to change to an appropriate return type?

Reply:
Since `runtime.internal.values.RegExpDisjunction` is not accessible by external users will deprecate and remove this.